kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -111,6 +111,7 @@ export default [
     props: route => {
       return {
         deviceId: route.params.deviceId,
+        goBackRoute: route.params.goBackRoute,
```

Comment:
Hrm, this doesn't seem quite right to me. The `goBackRoute` prop of the LibraryPage is a route object, but route params are strings, so I don't quite see how this is meant to be working.

Reply:
Ah, I see how this is kind of working - the underlying params object does get modified, but it doesn't get encoded in the URL. Unfortunately, it means if the page is refreshed the back link no longer works.

I think this is probably best handled in a similar way to the other back links in Learn, by encoding the name of the page in the query parameters.

Can add an additional function to this composable for creating the library page back links: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/composables/useContentLink.js#L7